### DISCUSSION TITLE
Keyboard and mouse support error

### DISCUSSION THREAD

**Discussion by PabloSebas** [redphx/better-xcloud#222] [General]
Good night. I'm opening this topic to report an error using keyboard and mouse support, after playing for a short time, the character begins to head in a specific direction as if that key were being pressed when in reality none were pressed.
The only solution found is to press F9 and activate again so that the character stops walking alone, i was looking on YouTube and I saw that several people have the same problem, it's a shame because the precision of the keyboard and mouse support is perfect, but with that detail unfortunately you can't play because it happens so often.
Thank you very much for reading the matter, greetings to the entire Xcloud community

**Comment by redphx** [comment]
Duplicate #220 
 
Fixed in 3.0.4
